### ISSUE TITLE
Getting Error: Cannot read property 'UNARY' of undefined

### ISSUE THREAD

**Issue by reselbob** [grpc/grpc-web#768]
Seem to be firing from Line 73 here:

https://github.com/reselbob/simple-node-grpc/blob/create-simple-web-client/client/js/simple_grpc_web_pb.js

The complete project is under the branch, `create-simple-web-client`

You can build the client environment by running the bash script here:

https://github.com/reselbob/simple-node-grpc/blob/create-simple-web-client/build/linux-build.sh

The gRPC server is in this directory: https://github.com/reselbob/simple-node-grpc/tree/create-simple-web-client/server

I am sort of at wit's end. If the error is due to my oversight, please forgive me and set me straight.

Still, I could use some help. If you can't, I understand completely.

Thanks in advance

**Comment by reselbob** [comment]
Let me add some information about the issue. 

It turns out that at line 75 of the following file:

https://github.com/reselbob/simple-node-grpc/blob/create-simple-web-client/client/js/simple_grpc_web_pb.js

`grpc.web.MethodType` is undefined

But `grpc.web` is an object.

Again, thanks in advance for any help.

**Comment by stanley-cheung** [comment]
Can you show us your `package.json` file? Which version of the `grpc-web` npm package did you end up pulling in?

**Comment by stanley-cheung** [comment]
Looks like you are pulling in `^0.4.0`. The `MethodType` class isn't defined then. You probably want something like `~1.0.7`.

**Comment by reselbob** [comment]
Thanks! That solved the problem, but now I am getting an error:

`Uncaught TypeError: SimpleService is not a constructor`

at line 6 in index.js

`const simpleService = new SimpleService('http://localhost:8080');`

**Comment by reselbob** [comment]
Solved it.

Seems that there is a naming convention in place in which the autogenerated code produces a name for the client based on the service name. Thus, in my `simple.proto` file I have a service named `SimpleService`. Thus, autogen-ing the web client create a class, SimpleServiceClient. I was unaware of this. I took a look a the sample code and saw, `EchoServiceClient`, so I figured what the heck. Adding the suffix, `Client` to the service name is the key.

Maybe I missed this naming convention in the documentation?
